uk_bin_collection/uk_bin_collection/councils/WarringtonBoroughCouncil.py

```
@@ -1,56 +1,48 @@
+    "Referer": "https://www.warrington.gov.uk/bin-collections",
+}
+
 
 class CouncilClass(AbstractGetBinDataClass):
     def parse_data(self, page: str, **kwargs) -> dict:
         user_uprn = kwargs.get("uprn")
         check_uprn(user_uprn)
-        bindata = {"bins": []}
 
         uri = f"https://www.warrington.gov.uk/bin-collections/get-jobs/{user_uprn}"
+        if not isinstance(schedule, list):
```

Ruff (0.15.17)

[error] Undefined name `schedule`

(F821)